deploy/roles/deploy.builder/templates/operators/elasticsearch/02-role.yaml

```diff
@@ -0,0 +1,66 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: elasticsearch-operator
+rules:
+- apiGroups:
+  - logging.openshift.io
+  resources:
+  - "*"
+  verbs:
+  - "*"
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - pods/exec
+  - services
+  - endpoints
+  - persistentvolumeclaims
+  - events
+  - configmaps
+  - secrets
+  - serviceaccounts
+  verbs:
+  - "*"
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  - daemonsets
+  - replicasets
+  - statefulsets
+  verbs:
+  - "*"
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
```

Comment:
So looking through the repo, I don't think it's doing anything that fancy. I think all it's intended to do is load in a set of prometheus rules and a service monitor so that metrics for elasticsearch itself can be setup to be monitored via the in-cluster monitoring in OpenShift.

At least this is my current working theory based on some artifacts I found in the elasticsearch-operator repository:

* https://github.com/openshift/elasticsearch-operator/blob/master/hack/prometheus-rules.yaml
* https://github.com/openshift/elasticsearch-operator/blob/021fa82b5c4f6fb0714dac00a5f5d0c793f7f7ce/pkg/k8shandler/service_monitor.go#L17
* https://github.com/openshift/elasticsearch-operator/blob/021fa82b5c4f6fb0714dac00a5f5d0c793f7f7ce/pkg/k8shandler/prometheus_rule.go#L65